renovate.json

```diff
@@ -0,0 +1,90 @@
+{
+   "extends":[
+      ":gitSignOff",
+      ":dependencyDashboard"
+   ],
+    "$schema":"https://docs.renovatebot.com/renovate-schema.json",
+    "baseBranches":[
```

Comment:
I've added a new `stable` branch on the release. Can you add it here, and in other places if needed?

Reply:
and for stable branch you would like only security updates or all updates?